parent/pom.xml

```diff
@@ -52,6 +52,7 @@
     <version.mockito>5.10.0</version.mockito>
     <version.wiremock>2.27.2</version.wiremock>
     <version.testcontainers>1.16.0</version.testcontainers>
+    <version.unirest-java>2.3.10</version.unirest-java>
```

Comment:
There already exists unirest-java-core in version 4.4.5. This requires another refactoring. Let's do this in a second iteration.